### PR TITLE
Support for ebs volume, fixed resource dependency

### PR DESCRIPTION
**Pull request by shashiranjan84**

### CODE REVIEW

File: roles/create_cf_stack/templates/coreos-beta-swarm-hvm.template [hunk 1]
```diff
@@ -574,7 +581,7 @@
           }
         ],
         "HealthCheck" : {
-          "Target" : { "Fn::Join" : [ "", ["HTTP:2379", "/health"]]},
+          "Target" : { "Fn::Join" : [ "", ["HTTP:4243", "/info"]]},
```
Comment (hutchic) [inline roles/create_cf_stack/templates/coreos-beta-swarm-hvm.template]: I think we want this to check `2379/health` which is etcd health no?

Comment (shashiranjan84) [reply roles/create_cf_stack/templates/coreos-beta-swarm-hvm.template]: i never thought of ETCD, assuming docker running on master node too, i changed it rely on docker port for health. Should i revert it back?

Comment (shashiranjan84) [reply roles/create_cf_stack/templates/coreos-beta-swarm-hvm.template]: @hutchic shall I add both ports to health check?

Comment (hutchic) [reply roles/create_cf_stack/templates/coreos-beta-swarm-hvm.template]: I guess both would work :+1:

**Pull request merged**